Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM alpine:3.13
 
-ARG KUBECTL_VERSION="1.21.2"
+ARG KUBECTL_VERSION="1.24.13"
 
 RUN apk add py-pip curl
 RUN pip install awscli
-RUN curl -L -o /usr/bin/kubectl https://amazon-eks.s3.us-west-2.amazonaws.com/1.21.2/2021-07-05/bin/linux/amd64/kubectl
+RUN curl -L -o /usr/bin/kubectl https://dl.k8s.io/v1.24.13/bin/linux/amd64/kubectl
```

Comment:
Don't we need an EKS version?

Reply:
It shouldn't make a difference but I'll change it.